### ISSUE TITLE
Store Bindings Information in ERC.yaml

### ISSUE THREAD

**Issue by NJaku01** [o2r-project/o2r-UI#224]
The Bindings Server now stores information about the Binding inside the ERC.yml. However this only works after the ERC.yml wac created by an execution. The ERC.yaml looks like this:
![image](https://user-images.githubusercontent.com/46575458/122061166-8c9de280-cdee-11eb-93d2-cb4bbfbe9f6e.png)

**Comment by Amritanshu02** [comment]
Hello! I would like to work upon this issue, could you suggest me where to start?

**Comment by nuest** [comment]
@Amritanshu02 Thanks for your interest -I think this one is quite complex and not a not first issue.

What is your background and interest in the o2r tools?

**Comment by nuest** [comment]
@NJaku01 I think the "after execution" part is actually not that big of a deal - we could add the step "save or update bindings" into the job steps.

Then we would also need to be able to _load_ the bindings information from this file, of course.
We should come up with a few test cases for `o2r-muncher`.

**Comment by Amritanshu02** [comment]
Hello!
I am a B.Tech- CSE 1st year student at Babu Banarasi Das Institute of
Technology and Management in Lucknow,UP, India.
I had just started with open source and I am looking for opportunities to
learn more about it and contribute.

**Comment by nuest** [comment]
Thanks for your interest @Amritanshu02 ! We are flattered you'd consider our project for your valuable time. However, there might be other, more suitable ones - would you mind letting me know what you are interested in, and what programming languages you'd like to work with?

**Comment by Amritanshu02** [comment]
Yes sure. I am interested in web development, Iot, cloud computing and many
more. I would like to work in java(intermediate level), HTML, CSS,
C(Basic).

**Comment by nuest** [comment]
@Amritanshu02 I'm really sorry but we do not have any projects that use Java. I recommend you look elsewhere, but I want to applaud your initiative. Here are some interesting projects that might work well for you from the context of scientific/geospatial software:

- https://worldwind.arc.nasa.gov/java/ (awesome project!)
- https://scijava.org/

I recommend to take a look at this nice guide, especially to find a project that you actually rely on, and then find a way to contribute to that: https://opensource.guide/how-to-contribute/

**Comment by Amritanshu02** [comment]
Thanks a lot! I will surely check them all.
